.travis.yml

```diff
@@ -1,19 +1,20 @@
 git:
   depth: 5
 language: node_js
+
 cache:
-  yarn: true
+  npm: false
+  directories:
+    - .yarn/cache
```

Comment:
I have experimented a bit with Yarn 2 on Travis CI. I came to conclusion that Travis cache is inefficient, it is faster to make Travis not cache anything, neither downloaded archives in `~/.yarn/berry` nor `node_modules`. I have got the best overall perfs with Travis, by setting:
```
cache: false
```
and using Yarn 2 settings below:
```
nodeLinker: node-modules
enableGlobalCache: true
compressionLevel: 0
```
`compressionLevel: 0` makes Yarn 2 store cache in zip files uncompressed, thus not spending much CPU time on compression. 
`enableGlobalCache: true` makes Yarn 2 store zip files only in one place inside `~/.yarn/berry/cache`, it normal y stores copy of them also in `<project_root>/.yarn/cache` via copy-on-write operation (which is not available on all the OSes).

You might have different results, just wanted to share some ideas with you.